.changeset/ten-guests-study.md

```diff
@@ -0,0 +1,5 @@
+---
+"@osdk/widget.api": minor
```

Comment:
Are we intending to backport all of these to release or wait to next release? Usual stuff about will need to change to patch on backport I think